internal/report/report.go

```diff
@@ -241,15 +242,25 @@ func (rpt *Report) newGraph(nodes graph.NodeSet) *graph.Graph {
 	for _, f := range prof.Function {
 		f.Filename = trimPath(f.Filename)
 	}
-	// Remove numeric tags not recognized by pprof.
+	// Remove numeric tags not recognized by pprof label with inferred units.
```

Comment:
This "not recognized by pprof label with inferred units." is not quite clear. What is "pprof label"? Can you rephrase?